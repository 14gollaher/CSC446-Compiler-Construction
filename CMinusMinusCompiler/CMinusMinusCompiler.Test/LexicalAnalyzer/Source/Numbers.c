123
123.45.12 1.
9 8 7.6
2.
cow 3.
cow3
if3
2147483647
2147483647.5
2if
_2
a2
               88
"88
9
"88"
"88
"88 8    
2.!