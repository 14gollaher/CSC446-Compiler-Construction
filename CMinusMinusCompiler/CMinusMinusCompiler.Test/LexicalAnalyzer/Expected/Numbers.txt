Lexeme                                 Token                          Attribute
-------------------------------------------------------------------------------
123                                    NumberToken                    123
123.45                                 NumberToken                    123.45
.                                      PeriodToken                    
12                                     NumberToken                    12
1                                      NumberToken                    1
.                                      PeriodToken                    
9                                      NumberToken                    9
8                                      NumberToken                    8
7.6                                    NumberToken                    7.6
2                                      NumberToken                    2
.                                      PeriodToken                    
cow                                    IdentifierToken                    
3                                      NumberToken					  3                    
.                                      PeriodToken                    
cow3                                   IdentifierToken    
if3									   IdentifierToken	
2147483647                             NumberToken					  2.147484E+09
2147483647.5						   NumberToken                    2.147484E+09
2                                      NumberToken                    2
if                                     IfToken   
_                                      UnknownToken                   
2                                      NumberToken                    2
a2                                     IdentifierToken     
88                                     NumberToken                    88
"88                                    UnknownToken                   
9                                      NumberToken                    9
"88"                                   StringLiteralToken             "88"
"88                                    UnknownToken      
"88 8                                  UnknownToken       
2                                      NumberToken                    2
.                                      PeriodToken                    
!                                      NotOperatorToken                   